s3/s3_operations.cpp

```diff
@@ -487,9 +523,12 @@ namespace irods_s3 {
             return std::make_tuple(PASS(ret), data.dstream_ptr, data.s3_transport_ptr);
         }
 
-        get_number_of_threads_data_size_and_opr_type(_ctx, number_of_threads, data_size, oprType);
+		ret = get_number_of_threads_data_size_and_opr_type(_ctx, number_of_threads, data_size, oprType);
+		if (!ret.ok()) {
+			return std::make_tuple(PASS(ret), data.dstream_ptr, data.s3_transport_ptr);
+		}
 
-        logger::debug("{}:{} ({}) [[{}]] oprType set to {}", __FILE__, __LINE__, __FUNCTION__, thread_id, oprType);
+		logger::debug("{}:{} ({}) [[{}]] oprType set to {}", __FILE__, __LINE__, __FUNCTION__, thread_id, oprType);
```

Comment:
See some tabs down here in this file

Reply:
I fixed that and a couple of other places.

There are still a couple of code segments that have tabs from previous pull requests/formats.  I can fix that later but don't want to confuse people by making those changes here.